backend/image/src/upload/responses/upload-image-conflict.error.ts

```diff
@@ -0,0 +1,9 @@
+import { ApiProperty } from "@nestjs/swagger";
+
+export class ImageUploadConflictError {
+    @ApiProperty({description: 'Message containing conflict reason'})
```

Comment:
In Errors ist das keine ApiResponseProperty?

Reply:
ApiResponsePropert erlaubt keine description. Deswegen benutz ich hier einfach nur property...und soweit ich das sehen kann macht das keinen Unterschied im generierten swagger 🤔